internal/provider/resource_auth0_custom_domain_test.go

```diff
@@ -47,35 +46,135 @@ func init() {
 	})
 }
 
-func TestAccCustomDomain(t *testing.T) {
-	if os.Getenv("AUTH0_DOMAIN") != recorder.RecordingsDomain {
```

Comment:
It turns out we didn't really need to run these tests only on the predefined recordings, we can run them on the live tenant on demand as well without anything breaking as we're not going through the verification.